.ci/install_nemu.sh

```diff
@@ -49,6 +52,35 @@ install_firmware() {
 	rm -f "${firmware}"
 }
 
-install_nemu
+install_prebuilt_nemu() {
+	install_directory="/tmp"
+	pushd "${install_directory}"
+	sudo -E curl -fL --progress-bar "${latest_build_url}/${NEMU_TAR}" -o "${NEMU_TAR}" || return 1
+	info "Install pre-built nemu version"
+	sudo tar -xvf "${NEMU_TAR}" -C /
+
+	info "Verify download checksum"
+	sudo -E curl -fsOL "${latest_build_url}/sha256sum-${NEMU_TAR}" || return 1
+	sudo sha256sum -c "sha256sum-${NEMU_TAR}" || return 1
+	popd
+}
+
+main() {
+	cached_nemu_version=$(curl -sfL "${latest_build_url}/latest") || cached_nemu_version="none"
+	info "current nemu : ${nemu_version}"
+	info "cached nemu  : ${cached_nemu_version}"
+	if [ "$cached_nemu_version" == "$nemu_version" ] && [ "$arch" == "x86_64" ]; then
```

Comment:
NEMU works on aarch64 too so this looks wrong to me.

Reply:
@jodh-intel we are only doing the installation of nemu on x86_64, we are not using it on aarch64, I talked with @chavafg and he agrees that this approach is correct

Reply:
How about adding a comment here explaining this as users might wonder why we're only testing on one arch when NEMU works on two?

- https://github.com/intel/nemu#nemu-a-cloud-hypervisor

Reply:
@jodh-intel changes applied, I added the comment